### ISSUE TITLE
Erroneous ERROR log in the broker daemon

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring/alignak#421]
The broker daemon log an ERROR when he gets its configuration for the schedulers:
```
[1475811259] ERROR: [broker-master] We have our schedulers: {u'aa18a25e8e9d4eea861b0a1c33c1fcad': {'broks': {}, u'data_timeout': 120, u'name': u'scheduler-master', 'last_connection': 0, u'hard_ssl_name_check': False, 'uri': u'http://127.0.0.1:7768/', u'instance_id': u'5f2442db4444488bbad18c672b043033', u'hosts': [u'windows_nsca_host', u'host_snmp', u'linux_nrpe', u'localhost', u'vm-win7', u'host_windows_wmi'], 'running_id': 0, u'timeout': 3, u'address': u'127.0.0.1', u'active': True, u'use_ssl': False, u'push_flavor': 868144, u'port': 7768}}
```

**Comment by mohierf** [comment]
Fixed with #451

**Comment by mohierf** [comment]
Closed with #453
